models/tasks.js

```diff
@@ -193,6 +194,10 @@ const fetchPaginatedTasks = async ({
         .where("title", "<=", title + "\uf8ff")
         .orderBy("title", "asc");
     }
+    if (assigneeRole === ROLES.ARCHIVED) {
+      const archivedUserIds = await userUtils.getArchivedUserIds();
+      initialQuery = initialQuery.where("assignee", "in", archivedUserIds);
```

Comment:
Firestore has a limit for 'in' operator. We cannot search for more than 30 user ids here. Please handle that case here.

Reply:
have handled it now please check